course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,164 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
```

Comment:
It's best to order your includes from the most specific to the most general ones. So in this case "my_string.h" should be first, and the STD headers last.